tests/utils/meilisearch-test-utils.ts

```diff
@@ -60,7 +60,7 @@ async function getClient(permission: string): Promise<MeiliSearch> {
     return publicClient
   }
 
-  if (permission === 'Private') {
+  if (permission === 'Admin') {
     const adminKey = await getKey(permission)
     const privateClient = new MeiliSearch({
```

Comment:
Can you change this name to `adminClient` ? One line below it should also be changed

Reply:
I have made this change